src/data/path.ts

```diff
@@ -13,65 +14,45 @@ export class ObjectsPath {
   buildCreate(consistencyLevel: string | undefined): Promise<string> {
     return this.build({ consistencyLevel }, [this.addQueryParams]);
   }
-  buildDelete(
-    id: string,
-    className: string,
-    consistencyLevel: string | undefined
-  ): Promise<string> {
+  buildDelete(id: string, className: string, consistencyLevel: string | undefined): Promise<string> {
     return this.build({ id, className, consistencyLevel }, [
       this.addClassNameDeprecatedNotSupportedCheck,
       this.addId,
       this.addQueryParams,
     ]);
   }
   buildCheck(id: string, className: string): Promise<string> {
-    return this.build({ id, className }, [
-      this.addClassNameDeprecatedNotSupportedCheck,
-      this.addId,
-    ]);
+    return this.build({ id, className }, [this.addClassNameDeprecatedNotSupportedCheck, this.addId]);
   }
   buildGetOne(
     id: string,
     className: string,
-    additionals: any,
-    consistencyLevel: string | undefined,
-    nodeName: any
+    additional: string[],
+    consistencyLevel?: ConsistencyLevel,
+    nodeName?: string
   ): Promise<string> {
-    return this.build(
-      { id, className, additionals, consistencyLevel, nodeName },
-      [
-        this.addClassNameDeprecatedNotSupportedCheck,
-        this.addId,
-        this.addQueryParams,
-      ]
-    );
+    return this.build({ id, className, additional: additional, consistencyLevel, nodeName }, [
+      this.addClassNameDeprecatedNotSupportedCheck,
+      this.addId,
+      this.addQueryParams,
+    ]);
   }
   buildGet(
     className: string | undefined,
```

Comment:
could be replaced with `className?: string`

Reply:
Yep, updated